### DISCUSSION TITLE
Add options for how shortend URLs are cleaned up

### DISCUSSION THREAD

**Discussion by lmickh** [grafana/grafana#71879] [General]
Before https://github.com/grafana/grafana/pull/61295, shortend URLs were only cleaned up if they were never used, https://github.com/grafana/grafana/pull/28867.  Now they are always deleted after 7 days.

Options should be added to either allow them to not be deleted if used and/or to set the time before they are deleted instead of hard coding it to 7 days.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424).  To move forward with this, I will be closing this discussion.

Short URLs should only be deleted after 7 days if they were never accessed. If that is not what you are experiencing, feel free to make an [issue](https://github.com/grafana/grafana/issues).
